packages/api-db-mongodb/src/db/comment.ts

```diff
@@ -48,12 +47,11 @@ export class MongoDBCommentAdapter implements DBCommentAdapter {
     this.locale = locale
   }
 
-  async createComment({input}: CreateCommentArgs): Promise<PublicComment> {
+  async addPublicComment({input}: AddPublicCommentArgs): Promise<PublicComment> {
     const {text, ...data} = input
     const {ops} = await this.comments.insertOne({
       ...data,
       state: CommentState.PendingApproval,
```

Comment:
This should also be set in the mutation. Certain users don't need to wait for approval.

Reply:
but for now, all of the comments will be pending approval yes?

Reply:
I have set this in the mutation instead of the adaptor and created a Jira ticket for handling the state depending on the author type.